pom.xml

```diff
@@ -77,7 +77,7 @@
                     <execution>
                         <id>attach-sources</id>
                         <goals>
-                            <goal>jar</goal>
+                            <!-- <goal>jar</goal> -->
```

Comment:
what is this change?

Reply:
Previously the build created 2 jars. One executable & One source jar. This disables the creation of the sources jar.